### PR TITLE
Add new topic for invoice payment events

### PR DESCRIPTION
**Pull request by danielcopaciu**

### CODE REVIEW

File: dev-aws/finance/topics.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+resource "kafka_topic" "inovice_payment_events" {
+  name               = "invoice.payment.events"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "retention.bytes" : "-1"
+    # keep for 30 days as it is indexed
+    "retention.ms" : "2592000000"
```
Comment (tosan88) [inline dev-aws/finance/topics.tf:14]: Do we want to rely on default values for the rest of the config?

Comment (danielcopaciu) [reply dev-aws/finance/topics.tf:14]: https://github.com/utilitywarehouse/kubernetes-manifests/blob/master/dev-aws/finance/kafka/kafka-topic-applier/topics.yaml#L319

Comment (danielcopaciu) [reply dev-aws/finance/topics.tf:14]: I copied the config from the old cluster. Which bit do you think is missing?

Comment (tosan88) [reply dev-aws/finance/topics.tf:14]: Fair enough, I only looked at `cleanup_policy: delete` missing

**Review by tosan88 — APPROVED** [review]

**Pull request merged**